src/dependency-registry.ts

```diff
@@ -379,42 +427,42 @@ export function handleSupportedCoreContractRemoved(
   coreContract.save();
 }
 
-export function handleProjectDependencyTypeOverrideAdded(
-  event: ProjectDependencyTypeOverrideAdded
+export function handleProjectDependencyOverrideAdded(
+  event: ProjectDependencyOverrideAdded
 ): void {
   const projectId = generateContractSpecificId(
-    event.params._coreContractAddress,
-    event.params._projectId
+    event.params.coreContractAddress,
+    event.params.projectId
   );
 
   const project = Project.load(projectId);
   if (!project) {
     return;
   }
 
-  project.scriptTypeAndVersion = event.params._dependencyType.toString();
+  project.scriptTypeAndVersion = event.params.dependencyNameAndVersion.toString();
```

Comment:
Also unrelated to this specific PR, but I don't think we should allow both core contract to set this field, and also the core registry. The state in the subgraph then depends on ordering, which might be pretty confusing? WDYT about a follow-on PR to do something like add a nullable `scriptTypeAndVersionOverride` field to project?

Reply:
Ah, yes definitely agree!